go.sum

```diff
@@ -49,8 +47,8 @@ github.com/creack/pty v1.1.9/go.mod h1:oKZEueFk5CKHvIhNR5MUki03XCEU+Q6VDXinZuGJ3
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c=
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
-github.com/dustinkirkland/golang-petname v0.0.0-20170105215008-242afa0b4f8a h1:ScNpctT+x1FcYMR+Oxij3AmQ8uDdwwDAHCPNo8OTc4U=
-github.com/dustinkirkland/golang-petname v0.0.0-20170105215008-242afa0b4f8a/go.mod h1:V+Qd57rJe8gd4eiGzZyg4h54VLHmYVVw54iMnlAMrF8=
+github.com/dustinkirkland/golang-petname v0.0.0-20191129215211-8e5a1ed0cff0 h1:90Ly+6UfUypEF6vvvW5rQIv9opIL8CbmW9FT20LDQoY=
```

Comment:
@detro I'm not 100% sure yet, but I'm seeing some issues after upgrading to 3.1.1 where I'm getting consistent petnames regardless of what I'm passing in for a keeper. If I downgrade to 3.1.0, it seems to work okay. Just flagging it for you in case anyone else writes in, but for now I'll try to get a repro case together and open an issue.

Reply:
Yes, unfortunately this was a non-innocuous update :(

The issue is tracked in #217 and I have opened a PR to fix it in #220.

Reply:
Thank you for reporting this @youcandanch